simulation/navigator_gazebo/launch/goose.launch

```diff
@@ -2,47 +2,425 @@
   <param name="use_sim_time" value="true" />
   <arg name="gui" default="true" />
 
-  <!-- start gazebo server-->
+
+  <!-- Gazebo Server -->
   <node name="gazebo" pkg="gazebo_ros" type="gzserver" respawn="false"
     args="$(find navigator_gazebo)/worlds/a_whole_new.world" output="screen"/>
   <node name="gazebo_gui" pkg="gazebo_ros" type="gzclient" respawn="false" if="$(arg gui)" output="screen" />
 
+
   <!-- El Boatatoe -->
   <node name="spawn_wamv" pkg="gazebo_ros" type="spawn_model"
     args="-file $(find navigator_gazebo)/models/wamv/wamv.sdf -sdf
       -x 0 -y 0 -z 1.5 -model wamv"
     output="screen" />
 
-  <!-- Shapes -->
-  <node name="spawn_blue_cross" pkg="gazebo_ros" type="spawn_model"
-    args="-file $(find navigator_gazebo)/models/shapes/blue_cross.sdf -sdf
-      -x 4 -y -18 -z 1.5 -model blue_cross"
+
+  <!-- Demonstrate Navigation and Control -->
+  <node name="spawn_speed_start_green" pkg="gazebo_ros" type="spawn_model"
```

Comment:
Can you put a flag in here so that gazebo can be launched like: `roslaunch navigator_launch simulation minimal:=true` and it won't spawn all the obstacles?